src/CommunityToolkit.Maui.Core/Essentials/Badge/IBadge.shared.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace CommunityToolkit.Maui.ApplicationModel;
+
+/// <summary>
+/// Badge
+/// </summary>
+public interface IBadge
+{
+	/// <summary>
+	/// Set the badge count
+	/// </summary>
+	/// <param name="count">Badge count</param>
+	void SetCount(int count);
+
+	/// <summary>
+	/// Get the badge count
+	/// </summary>
+	int GetCount();
```

Comment:
Any reason to choose a method instead of a property? From implementation on all platforms, this can be a property. I would say using a property is more a C# like

Reply:
That may be my fault. I recommend `int GetCount()` to Vlad as a corollary to `void SetCount()`. 

But I think you're right - as a dev, I'd rather just access a Property